src/clients/bridge.ts

```diff
@@ -1,5 +1,5 @@
 import { TextChannel } from 'discord.js';
-import { linkedListNode, nodeInterface } from '../linkedList';
+import { Node, LinkedList } from '../linkedList';
```

Comment:
Generally I think this is a good change.
I forget at this point, but I'm guessing the original reason I named it that way was so that it wasn't generic (node could be used for a lot of other scenarios). However, if we absolutely had to, we could easily just rename the import.

Probably the only thing I would change though if we do this is that we lowercase this and `LinkedList`, because all our other custom imports are lowercase as well. (for the sake of consistency)